pkg/ali/components/DiskGroup.vue

```diff
@@ -1,159 +1,125 @@
-    });
+const isCreate = computed(() => {
+  return props.mode === _CREATE;
+});
 
-    /**
+/**
      * Cleanup rows and emit input
      */
-    const update = () => {
-      if ( isView.value ) {
-        return;
-      }
-      const out = [];
-
-      for ( const row of rows.value ) {
-        const trim = !props.valueMultiline && (typeof row.value === 'string');
-        const value = trim ? row.value.trim() : row.value;
-
-        if ( typeof value !== 'undefined' ) {
-          out.push(value);
-        }
-      }
-      emit('update:value', out);
-    };
+const update = () => {
+  if ( isView.value ) {
+    return;
+  }
+  const out = [];
 
-    const lastUpdateWasFromValue = ref(false);
-    const queueUpdate = debounce(update, 50);
+  for ( const row of rows.value ) {
+    const value = row.value;
 
-    watch(
-      rows,
-      () => {
-        // lastUpdateWasFromValue is used to break a cycle where when rows are updated
-        // this was called which then forced rows to updated again
-        if (!lastUpdateWasFromValue.value) {
-          queueUpdate();
-        }
-        lastUpdateWasFromValue.value = false;
-      },
-      { deep: true }
-    );
+    if ( typeof value !== 'undefined' ) {
+      out.push(value);
+    }
+  }
+  emit('update:value', out);
+};
 
-    watch(
-      () => props.value,
-      () => {
-        lastUpdateWasFromValue.value = true;
-        rows.value = (props.value || []).map((v) => ({ value: v }));
-      },
-      { deep: true }
-    );
+const lastUpdateWasFromValue = ref(false);
+const queueUpdate = debounce(update, 50);
 
-    return {
-      rows,
-      lastUpdateWasFromValue,
-      queueUpdate,
-      isView,
-      isCreate,
-      update,
-    };
-  },
-  computed: {
-    disableAdd() {
-      return this.addDisabled;
-    },
-    showRemove() {
-      return this.removeAllowed;
+watch(
+  rows,
+  () => {
+    // lastUpdateWasFromValue is used to break a cycle where when rows are updated
+    // this was called which then forced rows to updated again
+    if (!lastUpdateWasFromValue.value) {
+      queueUpdate();
     }
+    lastUpdateWasFromValue.value = false;
   },
-  methods: {
-    add() {
-      this.rows.push({ value: clone(defaultAddValue) });
-      if (defaultAddValue) {
-        this.queueUpdate();
-      }
-      this.$nextTick(() => {
-        const inputs = this.$refs.value;
+  { deep: true }
+);
 
-        if ( inputs && inputs.length > 0 ) {
-          inputs[inputs.length - 1].focus();
-        }
-        this.$emit('add');
-      });
-    },
-    /**
-     * Remove item and emits removed row and its own index value
-     */
-    remove(row, index) {
-      this.$emit('remove', { row, index });
-      removeAt(this.rows, index);
-      this.queueUpdate();
-    },
+watch(
+  () => props.value,
+  () => {
+    lastUpdateWasFromValue.value = true;
+    rows.value = (props.value || []).map((v) => ({ value: v }));
+  },
+  { deep: true }
+);
+
+function add() {
```

Comment:
I see the code to focus one of the new inputs when a row is added was removed. Was that intentional? Looks like it doesn't work in `main` though so no a regression here